### PR TITLE
docs: Update changelog to 3.0

### PR DESCRIPTION
**Pull request by presatish**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,121 @@
...
+  BREAKING CHANGE: Commands via MessageBus topic configuration changed
+  ``` 
+- Remove ZeroMQ messagebus capability ([#848d5d6](https://github.com/edgexfoundry/device-onvif-camera/commits/848d5d6))
+  ```text
+  BREAKING CHANGE: Remove ZeroMQ messagebus capability
+  ```  
+  
+### Bug Fixes 🐛
+- Get command should return server error instead of timeout error ([#7f95bcf](https://github.com/edgexfoundry/device-onvif-camera/commits/7f95bcf))
+- Add ProfilesDir & DevicesDir to local config ([#309](https://github.com/edgexfoundry/device-onvif-camera/issues/309)) ([#b3088f7](https://github.com/edgexfoundry/device-onvif-camera/commits/b3088f7))
```
Comment (lenny-goodell) [inline CHANGELOG.md]: This can be remove as the bug was introduced in this cycle.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,121 @@
...
+- Change location of nats documentation ([#237](https://github.com/edgexfoundry/device-onvif-camera/issues/237)) ([#d65f0f9](https://github.com/edgexfoundry/device-onvif-camera/commits/d65f0f9))
+- Update docker compose download instructions to the latest version ([#223](https://github.com/edgexfoundry/device-onvif-camera/issues/223)) ([#81966fa](https://github.com/edgexfoundry/device-onvif-camera/commits/81966fa))
+- Updating validation metircs with  Hikvision camera ([#218](https://github.com/edgexfoundry/device-onvif-camera/issues/218)) ([#35c6aae](https://github.com/edgexfoundry/device-onvif-camera/commits/35c6aae))
+- Bump swagger generator version to 2.3.0 ([#198](https://github.com/edgexfoundry/device-onvif-camera/issues/198)) ([#12ac4ca](https://github.com/edgexfoundry/device-onvif-camera/commits/12ac4ca))
+- Bump Swagger version for Levski to 2.3.0 ([#197](https://github.com/edgexfoundry/device-onvif-camera/issues/197)) ([#9ddb556](https://github.com/edgexfoundry/device-onvif-camera/commits/9ddb556))
```
Comment (lenny-goodell) [inline CHANGELOG.md]: Remove. These don't make sense, from previous release...

**Review by lenny-goodell — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,121 @@
...
+- Remove docs ([#305](https://github.com/edgexfoundry/device-onvif-camera/issues/305)) ([#ca50ff0](https://github.com/edgexfoundry/device-onvif-camera/commits/ca50ff0))
+- Remove demo-app docs ([#273](https://github.com/edgexfoundry/device-onvif-camera/issues/273)) ([#30f142b](https://github.com/edgexfoundry/device-onvif-camera/commits/30f142b))
+- Add warning to main branch and link to levski ([#271](https://github.com/edgexfoundry/device-onvif-camera/issues/271)) ([#dcb5ff7](https://github.com/edgexfoundry/device-onvif-camera/commits/dcb5ff7))
+- Change location of nats documentation ([#237](https://github.com/edgexfoundry/device-onvif-camera/issues/237)) ([#d65f0f9](https://github.com/edgexfoundry/device-onvif-camera/commits/d65f0f9))
+- Update docker compose download instructions to the latest version ([#223](https://github.com/edgexfoundry/device-onvif-camera/issues/223)) ([#81966fa](https://github.com/edgexfoundry/device-onvif-camera/commits/81966fa))
+- Updating validation metircs with  Hikvision camera ([#218](https://github.com/edgexfoundry/device-onvif-camera/issues/218)) ([#35c6aae](https://github.com/edgexfoundry/device-onvif-camera/commits/35c6aae))
```
Comment (ChristianDarr-personal) [inline CHANGELOG.md]: Typo with `metircs`

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Pull request merged**